src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -108,18 +108,26 @@ def generate_weighted_strategy_pool():
   # If weighted strategy selection is enabled, there will be a distribution
   # stored in the environment.
   distribution = environment.get_value('STRATEGY_SELECTION_DISTRIBUTION')
+  selection_method = environment.get_value(
+      'STRATEGY_SELECTION_METHOD', default_value='default')
 
   # Otherwise if weighted strategy selection is not enabled (strategy selection
-  # distribution is none) or if we cannot query properly, generate strategy
+  # method is default) or if we cannot query properly, generate strategy
   # pool according to default parameters.
-  if not distribution:
+  if not distribution or selection_method == 'default':
     return generate_default_strategy_pool()
 
+  temperature = 'probability_medium_temperature'
```

Comment:
Nit: this variable name is confusing. It's actually a key used to get the probability for a given temperature.